### ISSUE TITLE
Exception in thread "main" com.github.dockerjava.api.BadRequestException

### ISSUE THREAD

**Issue by josephwinston** [ContainerSolutions/minimesos#472]
I've just followed the install instructions ```curl -sSL https://minimesos.org/install | sh``` and according to the ```~/.minimesos/bin/minimesos``` file, I have ```MINIMESOS_TAG="0.9.0"```. 

In an empty directory, I ran:

* [$ minimesos init](https://gist.github.com/josephwinston/f86495b3d41606b473e85ce8204ad26a#file-gistfile1-txt)
* [$ minimesos up](https://gist.github.com/josephwinston/c2d1e0b0e6bf13a2964c8331cda7d885#file-gistfile1-txt)

The generated traceback is:

```java
Exception in thread "main" com.github.dockerjava.api.BadRequestException: {"message":"starting container with HostConfig was deprecated since v1.10 and removed in v1.12"}

	at com.github.dockerjava.jaxrs.filter.ResponseStatusExceptionFilter.filter(ResponseStatusExceptionFilter.java:43)
	at org.glassfish.jersey.client.ClientFilteringStages$ResponseFilterStage.apply(ClientFilteringStages.java:134)
	at org.glassfish.jersey.client.ClientFilteringStages$ResponseFilterStage.apply(ClientFilteringStages.java:123)
	at org.glassfish.jersey.process.internal.Stages.process(Stages.java:171)
	at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:251)
	at org.glassfish.jersey.client.JerseyInvocation$1.call(JerseyInvocation.java:667)
	at org.glassfish.jersey.client.JerseyInvocation$1.call(JerseyInvocation.java:664)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:228)
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:424)
	at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:664)
	at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:424)
	at org.glassfish.jersey.client.JerseyInvocation$Builder.post(JerseyInvocation.java:333)
	at com.github.dockerjava.jaxrs.StartContainerCmdExec.execute(StartContainerCmdExec.java:28)
	at com.github.dockerjava.jaxrs.StartContainerCmdExec.execute(StartContainerCmdExec.java:13)
	at com.github.dockerjava.jaxrs.AbstrSyncDockerCmdExec.exec(AbstrSyncDockerCmdExec.java:23)
	at com.github.dockerjava.core.command.AbstrDockerCmd.exec(AbstrDockerCmd.java:33)
	at com.github.dockerjava.core.command.StartContainerCmdImpl.exec(StartContainerCmdImpl.java:53)
	at com.containersol.minimesos.container.AbstractContainer.start(AbstractContainer.java:93)
	at com.containersol.minimesos.cluster.MesosCluster.lambda$start$0(MesosCluster.java:126)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at com.containersol.minimesos.cluster.MesosCluster.start(MesosCluster.java:126)
	at com.containersol.minimesos.cluster.MesosCluster.start(MesosCluster.java:112)
	at com.containersol.minimesos.main.CommandUp.execute(CommandUp.java:104)
	at com.containersol.minimesos.main.Main.run(Main.java:125)
	at com.containersol.minimesos.main.Main.main(Main.java:56)
```

**Comment by adam-sandor** [comment]
This is caused by 0.9.0 not being compatible with Docker 1.12. @frankscholten just fixed this for 0.9.1 which we'll release in a few days.

**Comment by josephwinston** [comment]
Thank you. Please close.
